addons/hr_recruitment_skills/static/src/components/search_job_applicant_menu/search_job_applicant_menu.js

```diff
@@ -0,0 +1,53 @@
+/** @odoo-module **/
+
+import { Component, markup } from "@odoo/owl";
```

Comment:
odoo module isn't necessary anymore as it is a default (since 17.2 IIRC) 
```suggestion
import { Component, markup } from "@odoo/owl";
```